.github/workflows/file_health_exclusions_nonutf8.txt

```diff
@@ -0,0 +1,85 @@
+./dist/mac/CMakeLists.txt
+./dist/mac/qBitTorrentDocument.icns
+./dist/mac/qbittorrent_mac.icns
+./dist/qt-translations/qtbase_ar.qm
+./dist/qt-translations/qtbase_bg.qm
+./dist/qt-translations/qtbase_ca.qm
+./dist/qt-translations/qtbase_cs.qm
+./dist/qt-translations/qtbase_da.qm
+./dist/qt-translations/qtbase_de.qm
+./dist/qt-translations/qtbase_es.qm
+./dist/qt-translations/qtbase_fi.qm
+./dist/qt-translations/qtbase_fr.qm
+./dist/qt-translations/qtbase_gd.qm
+./dist/qt-translations/qtbase_he.qm
+./dist/qt-translations/qtbase_hu.qm
+./dist/qt-translations/qtbase_it.qm
+./dist/qt-translations/qtbase_ja.qm
+./dist/qt-translations/qtbase_ko.qm
+./dist/qt-translations/qtbase_lv.qm
+./dist/qt-translations/qtbase_pl.qm
+./dist/qt-translations/qtbase_ru.qm
+./dist/qt-translations/qtbase_sk.qm
+./dist/qt-translations/qtbase_uk.qm
+./dist/qt-translations/qtbase_zh_TW.qm
+./dist/qt-translations/qt_fa.qm
+./dist/qt-translations/qt_gl.qm
+./dist/qt-translations/qt_lt.qm
+./dist/qt-translations/qt_pt.qm
+./dist/qt-translations/qt_sl.qm
+./dist/qt-translations/qt_sv.qm
+./dist/qt-translations/qt_zh_CN.qm
+./dist/unix/menuicons/128x128/apps/qbittorrent.png
+./dist/unix/menuicons/128x128/status/qbittorrent-tray.png
+./dist/unix/menuicons/16x16/apps/qbittorrent.png
+./dist/unix/menuicons/16x16/status/qbittorrent-tray.png
+./dist/unix/menuicons/192x192/apps/qbittorrent.png
+./dist/unix/menuicons/192x192/status/qbittorrent-tray.png
+./dist/unix/menuicons/22x22/apps/qbittorrent.png
+./dist/unix/menuicons/22x22/status/qbittorrent-tray.png
+./dist/unix/menuicons/24x24/apps/qbittorrent.png
+./dist/unix/menuicons/24x24/status/qbittorrent-tray.png
+./dist/unix/menuicons/32x32/apps/qbittorrent.png
+./dist/unix/menuicons/32x32/status/qbittorrent-tray.png
+./dist/unix/menuicons/36x36/apps/qbittorrent.png
+./dist/unix/menuicons/36x36/status/qbittorrent-tray.png
+./dist/unix/menuicons/48x48/apps/qbittorrent.png
+./dist/unix/menuicons/48x48/status/qbittorrent-tray.png
+./dist/unix/menuicons/64x64/apps/qbittorrent.png
+./dist/unix/menuicons/64x64/status/qbittorrent-tray.png
+./dist/unix/menuicons/72x72/apps/qbittorrent.png
+./dist/unix/menuicons/72x72/status/qbittorrent-tray.png
+./dist/unix/menuicons/96x96/apps/qbittorrent.png
+./dist/unix/menuicons/96x96/status/qbittorrent-tray.png
+./dist/windows/nsis plugins/FindProcDLL_mod_by_hnedka.7z
+./dist/windows/nsis plugins/nsisFirewall.zip
+./dist/windows/nsis plugins/UAC.zip
+./.github/workflows/file_health_exclusions_cr.txt
```

Comment:
Why are all these text files non-UTF8?

Reply:
@glassez 

Same reasoning as https://github.com/qbittorrent/qBittorrent/pull/13659#discussion_r512657718 - because they're empty. If/when they stop being just empty files, we'll remove them from the exclusions.

Reply:
>If/when they stop being just empty files, we'll remove them from the exclusions.

Not so good reasoning... It can be simply forgotten.
>Because it's empty. With the current method (`file --mime`), empty files can't be considered UTF-8, regardless of their purpose or extension

Is it hard to check for file size before?

Reply:
fixed.